test/runtests.jl

```diff
@@ -102,7 +105,21 @@ using Test
 
         @test chain1[1].params == val
     end
+    
+    @testset "MALA" begin
+        
+        Sigma = 1e-2 * I(2)
+        
+        # Set up our sampler with initial parameters.
+        spl1 = MALA(x-> MvNormal((1/2) * Sigma * x, Sigma))
```

Comment:
This fails on Julia 1.0 since `I` is not callable. Instead you can use the equivalent formulation:
```suggestion      
        # Set up the sampler.
        sigma = 1e-1
        spl1 = MALA(x -> MvNormal((sigma^2 / 2) .* x, sigma))
```